src/commcare_cloud/ansible/roles/formplayer/templates/monit.formplayer.j2

```diff
@@ -2,4 +2,4 @@
   group formplayer
   start program = "supervisorctl start {{ project }}-{{ deploy_env }}-formsplayer-spring"
   stop program = "supervisorctl stop {{ project }}-{{ deploy_env }}-formsplayer-spring"
-  if content = "java.lang.OutOfMemoryError" then restart
\ No newline at end of file
+  if content = "java.lang.OutOfMemoryError: GC overhead limit exceeded" then restart
```

Comment:
I think Danny was saying this pattern should be anchored to the beginning of the line with a regex that positively matches a date pattern or something that is part of the logging output. Otherwise a user may be able to type the string "java.lang.OutOfMemoryError: GC overhead limit exceeded" in an input box and submit to trigger a restart.

Reply:
Yeah sorry for the lack of updates on the PR. I offlined with Danny about that on Friday and that is still in progress. This initial change addressed Cal's feedback.